.github/workflows/build-package.yaml

```diff
@@ -71,13 +65,11 @@ jobs:
           path: dist
 
       - name: Publish SDK package to pypi
-        run: |
-          if [[ $PYPI_HOST = "test.pypi.org" ]]; then
-            twine upload -r testpypi dist/* -u $PYPI_USER -p $PYPI_PASSWORD
-          else
-            twine upload dist/* -u $PYPI_USER -p $PYPI_PASSWORD
-          fi
-
+        uses: pypa/gh-action-pypi-publish@67339c736fd9354cd4f8cb0b744f2b82a74b5c70 # v1.12.3
+        if: ${{ secrets.PYPI_HOST == 'test.pypi.org' }}
```

Comment:
I tested on branch `[leo/test-trusted-publishing](https://github.com/openvinotoolkit/geti-sdk/tree/refs/heads/leo/test-trusted-publishing)`, it raises the error:

> [Invalid workflow file: .github/workflows/build-package-test-pypi.yaml#L14](https://github.com/openvinotoolkit/geti-sdk/actions/runs/13964312533/workflow)
The workflow is not valid. In .github/workflows/build-package-test-pypi.yaml (Line: 14, Col: 11): Error from called workflow openvinotoolkit/geti-sdk/.github/workflows/build-package.yaml@427f832592e6e8f8f2bfd5f072efe87700783d44 (Line: 69, Col: 13): Unrecognized named-value: 'secrets'. Located at position 1 within expression: secrets.PYPI_HOST == 'test.pypi.org'

Reply:
I think you should replace `secrets.PYPI_HOST` with `$PYPI_HOST`

Reply:
@leoll2 thanks for checking, I was slightly afraid of actual launching this script in an unfamiliar repo

Reply:
Thanks for the changes. I'm afraid the workflow is still invalid:

> [Invalid workflow file: .github/workflows/build-package-test-pypi.yaml#L14](https://github.com/open-edge-platform/geti-sdk/actions/runs/14079845057/workflow)
The workflow is not valid. In .github/workflows/build-package-test-pypi.yaml (Line: 14, Col: 11): Error from called workflow open-edge-platform/geti-sdk/.github/workflows/build-package.yaml@38c8ff21563ff75396712d86f22c6a8cd766cd9e (Line: 69, Col: 13): Unrecognized named-value: 'PYPI_HOST'. Located at position 1 within expression: PYPI_HOST == 'test.pypi.org'